Python/flowgraph.c

```diff
@@ -2239,7 +2239,7 @@ convert_pseudo_ops(basicblock *entryblock)
         for (int i = 0; i < b->b_iused; i++) {
             cfg_instr *instr = &b->b_instr[i];
             if (is_block_push(instr) || instr->i_opcode == POP_BLOCK) {
-                assert(SAME_OPCODE_METADATA(instr->i_opcode, NOP));
+                assert(OPCODE_HAS_ARG(instr->i_opcode) || instr->i_opcode == POP_BLOCK);
```

Comment:
Would it make sense to drop this assert and assert `OPCODE_HAS_ARG(instr->i_opcode)` in `is_block_push()`.
It seems a bit out of place here.